### ISSUE TITLE
2.0.0 says it's 1.1

### ISSUE THREAD

**Issue by peteraba** [zalando/zally#1195]
Short summary: zally 2.0.0 reports version 1.1.

**Steps to reproduce:**
1. Download zally_2.0.0_linux_amd64.tar.gz from https://github.com/zalando/zally/releases/tag/v2.0.0. (Likely works with other binaries as well)

2. Extract it.

2. Run zally without any arguments
```> ./zally 
NAME:
   Zally-CLI - Zally Command Line Interface

USAGE:
   zally [global options] command [command options] [arguments...]

VERSION:
```

**Expectation**
Version to contain "2.0.0"

**Actual**
Version contains "1.1"

**Comment by tkrop** [comment]
True, this is a bug in our release process that releases new CLI versions, while nothing has changed. Semantically v1.1 is correct but consistency requires that we would change the version, but we cannot (will not) update previous releases. Thanks for reporting, we will keep this inconsistency in mind for the next release.
